sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/parser/BeamSqlParserTest.java

```diff
@@ -163,13 +164,13 @@ private static Table mockTable(String name, String type, String comment, JSONObj
         .columns(ImmutableList.of(
             Column.builder()
                 .name("id")
-                .coder(INTEGER)
+                .typeDescriptor(TypeName.INT32.type())
                 .primaryKey(false)
                 .comment("id")
                 .build(),
             Column.builder()
                 .name("name")
-                .coder(VARCHAR)
+                .typeDescriptor(CalciteUtils.toFieldType(SqlTypeName.VARCHAR))
```

Comment:
<!--thread_id:cc_179026015_t; commit:9fe3a6d29a348e46ff2f5ca9e4396bf667e28046; resolved:0-->
<!--section:context-quote-->
> **akedin** wrote:
> I don't think that CalciteUtils should be used here, even if it's temporary. I'd rather have our own `SqlType.VARCHAR = TypeName.STRING.withMetadata("VARCHAR")`.

<!--section:body-->
Done. Added it to RowSqlTypes (so we can later remove the builder class there, but leave the defined types.